application/src/test/java/uk/nhs/hee/tis/revalidation/JsonSerializationTest.java

```diff
@@ -28,36 +28,24 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.junit.jupiter.MockitoExtension;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
-import org.springframework.boot.test.mock.mockito.MockBean;
-import uk.nhs.hee.tis.revalidation.controller.DoctorsForDBController;
 import uk.nhs.hee.tis.revalidation.dto.DoctorsForDbDto;
-import uk.nhs.hee.tis.revalidation.service.DoctorsForDBService;
-import uk.nhs.hee.tis.revalidation.service.RecommendationElasticSearchService;
 
-@ExtendWith(MockitoExtension.class)
-@WebMvcTest(DoctorsForDBController.class)
 class JsonSerializationTest {
 
-  @MockBean
-  private RecommendationElasticSearchService recommendationElasticSearchService;
-
-  @MockBean
-  private DoctorsForDBService mDoctorsForDbService;
-
-  @Autowired
-  private ObjectMapper mapper;
+  private final ObjectMapper mapper = (new RevalidationApplication()).objectMapper();
 
   @Test
   void testDateSerialization() throws JsonProcessingException {
```

Comment:
Honestly, do we need this test class at all? Fair enough if we feel the need to write integration tests for third party libraries but at this point it feels like we mayaswell be testing "toString" or "parseInt"